requirements.txt

```diff
@@ -1,3 +1,7 @@
 jupyterlab>=4.0.0,<5.0.0
 notebook<7
-reana-client @ git+https://github.com/mdonadoni/reana-client.git@vre-summer-24
\ No newline at end of file
+reana-client @ git+https://github.com/mdonadoni/reana-client.git@vre-summer-24
+pytest-asyncio
+pytest-mock
+pytest-jupyter
+pytest-jupyter[server]
```

Comment:
Why are these in the requirements? Are they used anywhere but in the pipeline?

Reply:
They are used to run the tests, nowhere else. I can install them in the workflow outside `requirements.txt` I think.